src/components/projects/A32NX/Download.tsx

```diff
@@ -5,7 +5,7 @@ export const Download: React.FC = () => {
     const urls = {
         stable: 'https://flybywiresim-packages.b-cdn.net/stable/A32NX-stable.zip',
         dev: 'https://flybywiresim-packages.b-cdn.net/vmaster/A32NX-master.zip',
-        exp: 'https://flybywiresim-packages.b-cdn.net/vmaster-cfbw-cap/A32NX-master-cfbw-cap.zip'
+        exp: 'https://flybywiresim-packages.b-cdn.net/vmaster-cfbw-cap/A32NX-master-cfbw-cap.zip',
```

Comment:
Adding extra commas where its not needed

Reply:
no we require trailing commas in the config

it's style

Reply:
deal with it

Reply:
bruh